pack.yaml

```
@@ -3,7 +3,7 @@ ref: vsphere
 name: vsphere
 description: VMware vSphere
 stackstorm_version: ">=2.9.0"
-version: 0.9.5
+version: 0.9.6
```

Since you've added a new feature it would be good to follow semver and bump the minor version to `0.10.10`. This review comment is a suggestion, so if you accept it or apply the suggestion it should automatically change it. 

```suggestion
version: 0.10.0
```